queries/dune_v2/period_slippage.sql

```diff
@@ -231,12 +231,13 @@ potential_buffer_trades as (
              token,
              block_time
              -- exclude 0 to prevent zero division, and exclude very small values for performance
-    having abs(sum(amount)) > 0.0001
-),
-valued_potential_buffered_trades as (
+    having sum(amount) != cast(0 as int256)
+)
+,valued_potential_buffered_trades as (
     select t.*,
-          amount * clearing_price            as clearing_value,
-          amount / pow(10, decimals) * price as usd_value
+    -- TODO: Not happy about this cast!
```

Comment:
Since were dealing with int256 here, is there any other alternative to this?

Reply:
I don't believe so -- because we are getting into prices (which are double)... In a perfect world we could use our own intrinsic (integer) prices, but this also comes along with some incompleteness.

Reply:
Thanks for the explanation. I'd suggest deleting the comment then.

Reply:
Done.